dbms/src/Storages/StorageKafka.h

```diff
@@ -53,21 +54,46 @@ friend class KafkaBlockOutputStream;
     void updateDependencies() override;
 
 private:
+    /// Each engine typically has one consumer (able to process 1..N partitions)
+    /// In the future the table engine could have multiple consumers for better throughput
```

Comment:
We have support for multiple consumers right now?

Reply:
Yes, I ended up adding it. Let me update the comment.